### ISSUE TITLE
Add maturity assessment worksheet

### ISSUE THREAD

**Issue by afeijoo** [uswds/uswds-site#934]
- [x] QA the [USWDS maturity assessment worksheet](https://drive.google.com/file/d/1rZjMlD-W3wKApxj-GzsMmNQMKnbBKy3S/view):lock:
- [x] Add card component in https://github.com/uswds/uswds-site/issues/950
- [x] Confirm 508 for PDF; see [accessibility report](https://drive.google.com/file/d/1BXON7MjQeQ4nw4i-pjWXSoUWeGJIM2i2/view?ts=5ec2dd5b) (cc: @nks-bix)
- [x] Draft [new website copy](https://docs.google.com/document/d/1Gs5auRCBN4nX2EqHiXa90ZxVyz8EM1HBTW-Le759qGQ/edit#):lock:
- [ ] Update site copy to integrate w/site at https://designsystem.digital.gov/maturity-model/#maturity-assessment-resources

**Comment by KimberlyHodges** [comment]
@afeijoo Card is not listed in the Component section. Do you want to add it before you publish this?

**Comment by afeijoo** [comment]
@KimberlyHodges The card isn't released yet, so let's work in parallel. I added a new issue to capture adding the card component to the worksheet: https://github.com/uswds/uswds-site/issues/950

**Comment by KimberlyHodges** [comment]
@nks-bix I think you can forge ahead with having Meg review the PDF for 508 Compliance.

**Comment by KimberlyHodges** [comment]
@afeijoo I've completed draft review of doc and it looks good other than adding #950 card component. Awaiting completion of 508 check, which @nks-bix is driving. Next step to determine where to integrate doc within design system website.

**Comment by KimberlyHodges** [comment]
@mejiaj and @afeijoo here's link to google doc with my guidance on how to integrate with site: [https://docs.google.com/document/d/1Gs5auRCBN4nX2EqHiXa90ZxVyz8EM1HBTW-Le759qGQ/edit?usp=sharing
](url)

**Comment by thisisdano** [comment]
Updated the Worksheet with newest components and guidance: [Worksheet v0.2](https://drive.google.com/drive/folders/1c0XyFpMfpQHG6uKFp1_1aWGx0esBBR9w)

**Comment by nks-bix** [comment]
confirmed on Accessibility for the pdf.  https://drive.google.com/open?id=1BXON7MjQeQ4nw4i-pjWXSoUWeGJIM2i2

**Comment by afeijoo** [comment]
@mejiaj Can you please update https://designsystem.digital.gov/maturity-model/#maturity-assessment-resources with the [new site copy](https://docs.google.com/document/d/1Gs5auRCBN4nX2EqHiXa90ZxVyz8EM1HBTW-Le759qGQ/edit#) to add the [worksheet v0.2](https://drive.google.com/drive/folders/1c0XyFpMfpQHG6uKFp1_1aWGx0esBBR9w)?
